test/service/test_tdr.py

```diff
@@ -369,7 +371,7 @@ def _bq_schema(self, fields: Mapping[str, Union[str, type]] = frozendict(), **kw
         kwargs.update(fields)
         return [
             dict(name=k,
-                 type=self._bq_types.get(v, v),
+                 type=(self._bq_types[v] if isinstance(v, type) else v),
```

Comment:
were the outer parenthesis intentional here?

Reply:
Yes, I acknowledge that they are semantically superfluous but my personal experience is that they make the line easier to visually scan.